components/post_view/post/post.jsx

```diff
@@ -163,8 +163,11 @@ export default class Post extends React.PureComponent {
             rootUser = '';
         }
 
-        if (isSystemMessage) {
+        if (isSystemMessage || isMeMessage) {
```

Comment:
Nit 1/5 - instead of duplicating isSystemMessage check, maybe do:
```suggestion
  if (isSystemMessage || isMeMessage) {
        if(isMeMessage) className += ' post--system';	                   
            sameUserClass = '';	            
            currentUserCss = '';	        
            postType = '';	            
   }
```

Reply:
This was my thought process:

I can either duplicate the assignment of the additional class name of `post--system` with two seperate if checks like so:
```suggestion
if (isMeMessage) {
    className += ' post--system';
}

if (isSystemMessage) {
    className += ' post--system';
    sameUserClass = '';
    currentUserCss = '';
    postType = '';
}
```

Or, I do it the way it is currently implemented by DRYing up the class name concatenation into one if statement.

I would argue that for now since /me messages are sharing system message styling, that having that concatenation in one place, rather than two, structures it in the way that makes the most sense.

Counter Nit (Nitpick?, was't sure what that meant haha) - The suggested change would actually introduce a bug that does not attach the correct styling to system messages.

` if (isMeMessage) className += ' post--system'` would only attach the class that applies system styling to me messages and not system messages.

Reply:
I would've actually said they could be kept separate since the original intent of this was to provide different styling for `/me` messages, but I'm fine with either way of doing this

Reply:
I thought about that is well but that would be simple to refactor given that more "/me" message specific logic was introduced.